CMakeLists.txt

```
@@ -39,6 +39,15 @@ else()
 endif()
 # ------------------
 
+# ------------------
+# Extensive mode for running extension oneAPI sub_group_mask tests
+option(SYCL_EXT_ONE_API_ENABLE_SUB_GROUP_MASK_TESTS
```

```suggestion
option(SYCL_ENABLE_EXT_ONEAPI_SUB_GROUP_MASK_TESTS
```
- ONE_API -> ONEAPI 
- Don't put ENABLE in the middle of the extension name.

---

This comment was replaced in this commit: a294d1910b53c2fb50a3ded0e4aa10907dcd5cf2